conda_pack/tests/test_core.py

```diff
@@ -123,15 +123,11 @@ def test_env_properties(py36_env):
 
 def test_load_environment_ignores(py36_env):
     lk = {normpath(f.target): f for f in py36_env}
-
-    for path in ['{}/conda'.format(BIN_DIR_L), 'conda-meta']:
-        assert path not in lk
-
-    # activate/deactivate exist, but aren't from conda
-    bat_suffix = '.bat' if on_win else ''
-    for file in ('activate', 'deactivate'):
-        fname = '{}/{}{}'.format(BIN_DIR_L, file, bat_suffix)
-        assert not lk[fname].source.startswith(py36_path)
+    for fname in ('conda', 'conda.bat'):
+        assert '{}/{}'.format(BIN_DIR_L, fname) not in lk
```

Comment:
Shouldn't this be `os.path.join`? the `normpath` above converts windows paths to use backslashes, so this will always pass?

Reply:
Same for below.

Reply:
I'll double check this, thanks. I didn't appreciate that subtlety of normpath.

Reply:
After looking at this with @mcg1969, we noticed that `normpath` was defined in this file (and is not the `os.path.normpath` that I thought it was). This is confusing, but does mean that the test is correct in its current form.

I'm going to merge this as is, as I'd like to issue a release. At some point in the future we should clean up the path handling in the test suite to make this behavior clearer (or at least rename `normpath` here to something that doesn't name-clash with a common routine in the standard library).